src/actions/index.js

```diff
@@ -15,7 +15,7 @@ export const fetchRecentSnippets = marker => (dispatch) => {
   let qs = ''
   if (marker) { qs = `&marker=${marker}` }
 
-  return fetch(misc.getApiUri(`/snippets?limit=20${qs}`))
```

Comment:
I'd personally prefer to preserve leading `/` in passed endpoint, but it does not matter much. :)